docs/usage.md

```diff
@@ -0,0 +1,59 @@
+---
+description: Usage hints for DietPi
+title: Usage hints / HowTo
+---
+
+# Usage hints / HowTo
+
+This chapter contains several documents describing the usage of DietPi.
+
+## How to use the logging mechanism
+
+DietPi uses the systemd based logging (instead of using syslog). `journald` is the logging part.
+The basic command to evaluate the systemd-journal is
+
+```sh
+journalctl [options]
+```
+
+<font size="+2">Basic output</font>
```

Comment:
What do you think about making those real headings (as it's not part of a tab)? When more topics are added, the text should contain the section name somehow, like:
```
### Logging output

### Logging filtering

### Logging options
```

Reply:
@MichaIng:
I started with this. But my idea was then to keep them away from the navigation on the right side. In case of about >20 HowTo sections the navigation tree would become too large.  
In case of 10 HowTos with each of 3 subheadings we would get 40 navigation lines (which cannot be collapsed).  
The other idea could be to have a tree on the right side of the navigation tree (like the "Optimized Software" subtree). Then every HowTo would be one sub entry on the left side and the navigation within the HowTo (with its three entries in this example) would appear on the right side.

What do you think about this?

Reply:
Ah I see and remember you mentioned that. I think when the amount of HowTo's grows above a certain limit, indeed we'd need to split the page anyway (making HowTo's a new primary navigation entry, as you suggested) or otherwise move content into tabs to have the page size + ToC not too large. But we discuss the downsides of tabs for software docs (like not searchable, also still part of DOM/file size), so I'd currently go with splitting the page then. Splitting could also be done by topic/category, probably.

So the question is if we want to avoid headings already now, or remove them if/once the ToC has grown too large?

Reply:
Discussion before: Avoid headings now, change headings names with "logging" keyword.